bluesky/ui/qtgl/glhelpers.py

```diff
@@ -915,10 +918,11 @@ class UniformBufferObject(GLBuffer):
     ufo_max_binding = 1
 
     def __init__(self):
-        super().__init__(gl.GL_UNIFORM_BUFFER)
+        # super().__init__(gl.GL_UNIFORM_BUFFER)
+        super().__init__(QOpenGLBuffer.Type.VertexBuffer)
```

Comment:
Very unsure about what I did here

Reply:
I could not find a uniform buffer type [here](https://www.riverbankcomputing.com/static/Docs/PyQt6/api/qtopengl/qopenglbuffer.html?highlight=qopenglbuffer#QOpenGLBuffer).

![image](https://user-images.githubusercontent.com/78442543/148531631-360cc5d9-ab5d-4c72-9bf8-ea961e5be4dc.png)